repos/fdbt-site/src/pages/api/salesConfirmation.ts

```diff
@@ -49,7 +49,12 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
             userDataJson = getSingleTicketJson(req, res);
         } else if (fareType === 'return') {
             userDataJson = getReturnTicketJson(req, res);
-        } else if (fareType === 'period' || fareType === 'multiOperator' || fareType === 'flatFare') {
+        } else if (
+            fareType === 'period' ||
```

Comment:
can we combine and use in operator like

fareType in ['period', ...]

Reply:
['period', 'multiOperator', 'flatFare', 'cappedProduct'].includes(fareType)

Reply:
probably will do this now